lib/brain.js

```diff
@@ -760,6 +761,10 @@ Brain.prototype._processReal = function _processReal (req) {
       return this.lightningMode(true)
     case 'lightningNo':
       return this.lightningMode(false)
+    case 'termsAccepted':
+      this.fastUpdateTx({termsAccepted: true})
```

Comment:
The problem here is that the tx is not yet ready for sending to the server, as there are missing fields leading to a server error. Since we don't actually have to send the tx to the server yet, we should replace with something closer to your original change:

```
      this.tx = Tx.update(this.tx, {termsAccepted: true})
```

Reply:
got it, I would put those in a function thought